lib/hammer_cli_katello/repository.rb

```diff
@@ -219,7 +219,7 @@ class UpdateCommand < HammerCLIKatello::UpdateCommand
       option "--publish-via-http", "ENABLE", _("Publish Via HTTP"),
              :attribute_name => :option_unprotected,
              :format => HammerCLI::Options::Normalizers::Bool.new
-      option "--docker-tag", "TAG", _("Container Image tag")
+      option "--docker-tag", "TAG", _("Optional Container Image tag to specify Docker digest")
```

Comment:
Do you not want to match the documentation wording?

Reply:
I thought that the line was too long? Sorry this is the first time submitting a PR for this project. I think my original line is better, but if it fails validation, what can I do?

Reply:
I showed in my first comment how to achieve what you want. You can just continue the statement on the next line, like this:

```suggestion
      option "--docker-tag", "TAG",
        _("Optional custom Container Image tag to specify the value of the Docker digest")
```

Reply:
I'm so thick!!